modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/06_create_new_app/page-definition.json

```diff
@@ -0,0 +1,13 @@
+{
+	"pageElement": {
+		"pageElements": [
+		],
```

Comment:
Don't we need to put the client extension here?

Reply:
Fixed